multinode_integration_tests/tests/verify_bill_payment.rs

```diff
@@ -1,62 +1,38 @@
 // Copyright (c) 2019, MASQ (https://masq.ai) and/or its affiliates. All rights reserved.
-use bip39::{Language, Mnemonic, Seed};
-use futures::Future;
-use itertools::Either;
+
+use crate::verify_bill_payment_utils::utils::{
+    test_body, to_wei, AssertionsValues, Debt, DebtsSpecs, FinalServiceFeeBalancesByServingNodes,
+    NodeProfile, Ports, TestInputsBuilder, WholesomeConfig,
+};
+use itertools::Itertools;
 use masq_lib::blockchains::chains::Chain;
 use masq_lib::messages::FromMessageBody;
 use masq_lib::messages::ToMessageBody;
 use masq_lib::messages::{ScanType, UiScanRequest, UiScanResponse};
-use masq_lib::percentage::Percentage;
-use masq_lib::utils::{derivation_path, find_free_port, NeighborhoodModeLight};
-use multinode_integration_tests_lib::blockchain::BlockchainServer;
+use masq_lib::percentage::PurePercentage;
+use masq_lib::utils::find_free_port;
 use multinode_integration_tests_lib::masq_node::MASQNode;
 use multinode_integration_tests_lib::masq_node_cluster::MASQNodeCluster;
-use multinode_integration_tests_lib::masq_real_node::{
-    ConsumingWalletInfo, EarningWalletInfo, MASQRealNode, NodeStartupConfig,
-    NodeStartupConfigBuilder,
-};
-use multinode_integration_tests_lib::utils::{
-    node_chain_specific_data_directory, open_all_file_permissions, UrlHolder,
-};
-use node_lib::accountant::db_access_objects::payable_dao::{PayableDao, PayableDaoReal};
-use node_lib::accountant::db_access_objects::receivable_dao::{ReceivableDao, ReceivableDaoReal};
-use node_lib::accountant::db_access_objects::utils::to_time_t;
-use node_lib::accountant::gwei_to_wei;
-use node_lib::blockchain::bip32::Bip32EncryptionKeyProvider;
-use node_lib::blockchain::blockchain_interface::blockchain_interface_web3::{
-    BlockchainInterfaceWeb3, REQUESTS_IN_PARALLEL,
-};
-use node_lib::blockchain::blockchain_interface::BlockchainInterface;
-use node_lib::database::db_initializer::{
-    DbInitializationConfig, DbInitializer, DbInitializerReal, ExternalData,
-};
+use multinode_integration_tests_lib::masq_real_node::{MASQRealNode, NodeStartupConfigBuilder};
 use node_lib::sub_lib::accountant::PaymentThresholds;
-use node_lib::sub_lib::blockchain_interface_web3::{
-    compute_gas_limit, transaction_data_web3, web3_gas_limit_const_part,
-};
-use node_lib::sub_lib::wallet::Wallet;
-use node_lib::test_utils;
-use rusqlite::ToSql;
-use rustc_hex::{FromHex, ToHex};
+use node_lib::sub_lib::blockchain_interface_web3::{compute_gas_limit, web3_gas_limit_const_part};
 use std::convert::TryFrom;
-use std::path::{Path, PathBuf};
-use std::time::{Duration, Instant, SystemTime};
-use std::{thread, u128};
-use tiny_hderive::bip32::ExtendedPrivKey;
-use web3::transports::Http;
-use web3::types::{Address, Bytes, SignedTransaction, TransactionParameters, TransactionRequest};
-use web3::Web3;
+use std::time::Duration;
+use std::u128;
+
+mod verify_bill_payment_utils;
 
 #[test]
-fn verify_bill_payment() {
+fn full_payments_were_processed_for_sufficient_balances() {
```

Comment:
You'd like to rename it to `full_payments_were_processed_as_balances_were_sufficient()`.